reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md

```diff
@@ -9,101 +9,129 @@ title:  Set-Acl
 
-The first command uses the Get-Acl cmdlet to get the security descriptor of the Dog.txt file.
-The assignment operator (=) stores the security descriptor in the value of the $DogACL variable.
+The second command uses **Set-Acl** to change the values in the ACL of Cat.txt to the values in
+$DogACL.
 
-The second command uses **Set-Acl** to change the values in the ACL of Cat.txt to the values in $DogACL.
+The value of the **Path** parameter is the path to the Cat.txt file. The value of the **AclObject**
+parameter is the model ACL, in this case, the ACL of Dog.txt as saved in the $DogACL variable.
 
-The value of the **Path** parameter is the path to the Cat.txt file.
-The value of the **AclObject** parameter is the model ACL, in this case, the ACL of Dog.txt as saved in the $DogACL variable.
 ### Example 2
+
 ```
 PS C:\> Get-Acl C:\Dog.txt | Set-Acl -Path C:\Cat.txt
 ```
 
-This command is almost the same as the command in the previous example, except that it uses a pipeline operator to send the security descriptor from a Get-Aclcommand to a **Set-Acl** command.
+This command is almost the same as the command in the previous example, except that it uses a
+pipeline operator to send the security descriptor from a Get-Aclcommand to a **Set-Acl** command.
 
 The first command uses the **Get-Acl** cmdlet to get the security descriptor of the Dog.txt file.
-The pipeline operator (|) passes an object that represents the Dog.txt security descriptor to the **Set-Acl** cmdlet.
+The pipeline operator (|) passes an object that represents the Dog.txt security descriptor to the
+**Set-Acl** cmdlet.
 
 The second command uses **Set-Acl** to apply the security descriptor of  Dog.txt to Cat.txt.
 When the command completes, the ACLs of the Dog.txt and Cat.txt files are identical.
+
 ### Example 3
+
 ```
 PS C:\> $NewAcl = Get-Acl File0.txt
 PS C:\> Get-ChildItem c:\temp -Recurse -Include *.txt -Force | Set-Acl -AclObject $NewAcl
 ```
 
-These commands apply the security descriptors in the File0.txt file to all text files in the C:\Temp directory and all of its subdirectories.
+These commands apply the security descriptors in the File0.txt file to all text files in the
+C:\Temp directory and all of its subdirectories.
 
-The first command gets the security descriptor of the File0.txt file in the current directory and uses the assignment operator (=) to store it in the $NewACL variable.
+The first command gets the security descriptor of the File0.txt file in the current directory and
+uses the assignment operator (=) to store it in the $NewACL variable.
 
-The first command in the pipeline uses the Get-ChildItem cmdlet to get all of the text files in the C:\Temp directory.
-The **Recurse** parameter extends the command to all subdirectories of C:\temp.
+The first command in the pipeline uses the Get-ChildItem cmdlet to get all of the text files in the
+C:\Temp directory. The **Recurse** parameter extends the command to all subdirectories of C:\temp.

```

Comment:
When editing files, can you please add semantic linebreaks? http://rhodesmill.org/brandon/2012/one-sentence-per-line/ basically a linebreak after the end of each sentence. This makes the diffs a lot easier when we read them in the future

Reply:
@zjalexander Will do. My editor is configured to break lines at 100 columns.

Reply:
Yes, please. I didn't get semantic line breaks until I started doing some longer form diffs/reviews. Makes life so much easier :)

Reply:
Do you want me to re-edit these?